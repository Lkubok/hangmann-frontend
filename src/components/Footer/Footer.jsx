import React, { Component } from "react";
import "./Footer.scss";

export default class Footer extends Component {
  render() {
    return (
      <div className="app-footer">
        {/* <img className="hang-image" src="./img/hanged-man.png" alt="" /> */}
      </div>
    );
  }
}
