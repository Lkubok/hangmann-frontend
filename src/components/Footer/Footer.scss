@import "../../scss/mixins.scss";
@import "../../scss/variables.scss";


.app-footer {
    @include basic-app-padding;
    @include flex-cl-row;
    @include basic-app-margin;
    background-color: $dark-color;
    color: $white-color;
    min-height: 250px;
    padding-bottom: 50px;
    background-image: url(/img/hanged-man.png);
    background-repeat: repeat-x;
    border-top: 5px solid $black-color;
    @media all and (max-width: 992px) {
        min-height: 225px;
      padding-bottom: 5px;
      }
  }
  