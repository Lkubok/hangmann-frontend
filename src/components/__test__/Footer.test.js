import React from "react";
import Enzyme, { shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import { Footer } from "../Footer/Footer";
Enzyme.configure({ adapter: new Adapter() });

describe("Footer component", () => {
  beforeEach(() => {});
  it("Should match snapshot", () => {
    const wrapper = shallow(<Footer />);
    expect(wrapper).toMatchSnapshot();
  });
});
