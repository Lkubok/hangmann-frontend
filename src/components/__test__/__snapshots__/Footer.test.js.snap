// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component Should match snapshot 1`] = `
<div
  className="app-footer"
/>
`;
